jest.config.ts

```diff
@@ -0,0 +1,42 @@
+import type {Config} from '@jest/types';
```

Comment:
Nit: there should be spaces around `Config` e.g. `{ Config }` - I think this is due to your VS Code not auto-formatting on save. Please check my above comment.

Reply:
Again, think we need prettier if we want to do this. Or a linter that errors out on the PR (ideally both)